CHANGELOG.md

```diff
@@ -43,6 +43,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed `$pkg.metadata.get_payload_format()`. It is still possible to fetch manually, but
   practically speaking it is not meaningful. rpmbuild has written a misleading value here for
   10 years.
+- Added `CompressionType`
```

Comment:
This isn't a breaking change, move it up to the `Added` section

Reply:
Ups, thank you! Done.